src/ossia-max/src/remote.cpp

```diff
@@ -90,29 +98,43 @@ void* remote::create(t_symbol* name, long argc, t_atom* argv)
     // process attr args, if any
     attr_args_process(x, argc - attrstart, argv + attrstart);
 
-    if (x->m_name != _sym_nothing)
-    {
-      x->update_path();
-      ossia_check_and_register(x);
-    }
+    // Register object to istself so it can receive notification when attribute changed
+    // This is not documented anywhere, please look at :
+    // https://cycling74.com/forums/notify-when-attribute-changes
+    object_attach_byptr_register(x, x, CLASS_BOX);
 
-    ossia_max::instance().remotes.push_back(x);
+    // need to schedule a loadbang because objects only receive a loadbang when patcher loads.
+    x->m_reg_clock = clock_new(x, (method) object_base::loadbang);
```

Comment:
since this is used in a few places maybe it'd make sense to refactor in a small init_and_bang_clock() (or something like that) method in an ossia base class  ?

Reply:
sure, I've put a TODO for the next refactoring